UI/BuildDesignatorWnd.cpp

```diff
@@ -1051,7 +1051,8 @@ void BuildDesignatorWnd::CenterOnBuild(int queue_idx, bool open) {
             MapWnd* map = ClientUI::GetClientUI()->GetMapWnd();
             map->CenterOnObject(system_id);
             if (open) {
-                SelectSystem(system_id);
+                HumanClientApp::GetApp()->GetClientUI().GetMapWnd()->SelectSystem(system_id);
+                //SelectSystem(system_id);
```

Comment:
Wow, that is quite a lecture for what I already said was accidental-- a simple "Yes, I was referring to the commented line" seems it would have been 100% sufficient.   Are you trying to say you deem it to be bad practice to comment out code while working on replacing it?  That the code must be fully deleted **before** the replacement code is ready and working?